src/components/auth/torus/AuthTorus.js

```diff
@@ -298,23 +297,6 @@ const AuthTorus = ({ screenProps, navigation, styles }) => {
     }
   }
 
-  useEffect(() => {
```

Comment:
this was needed for Torus redirect flow
why have you removed it ?

Reply:
old login flow

Reply:
@roma-hladilka please re-check does it works correctly on web and android without this

Reply:
will do

Reply:
@johnsmith-gooddollar Do we need to show error "We were unable to load the wallet", if user canceled login, as it works now on android? Maybe we need to add the condition to not show the error in case of cancel?

Reply:
Regarding useeffect, it works on web and android

Reply:
if user cancelled login he/she should be redirected to the welcome